specification/applicationinsights/data-plane/microsoft.insights/preview/2018-04-20/swagger.json

```diff
@@ -0,0 +1,2071 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-04-20",
```

Comment:
We're actually keeping both, I realize this is a bit funky. We're data plane but accessible via both ARM and a direct endpoint. For the direct endpoint (e.g. api.loganalytics.io), we use our own versioning (e.g. "v1"). In parallel we offer versions using Azure-style date string accessible via management.azure.com. 

Generally these are very close/almost identical to their direct counterparts except where parameters differ, e.g. taking the Azure resource ID vs. taking a service-specific GUID. We doc the URL syntax (with versions) for both. Besides the URLs, the only difference is that the direct endpoint supports substantially larger payloads than ARM's limits (which matters, since we often return large DB query results).